### ISSUE TITLE
ERROR - server failed: Error: invalid blockchain info

### ISSUE THREAD

**Issue by duckartes** [romanz/electrs#10]
Caused by: invalid value: integer `198853069730`, expected usize

Not sure why it's throwing this. Node is current.

**Comment by romanz** [comment]
Thanks for reporting this issue!
AFAIU, it should be fixed by https://github.com/romanz/electrs/commit/53c0c02732f4556bfe90feaeda1213329e39e99d - please take a look :)

**Comment by romanz** [comment]
BTW, are you running this code by chance on a 32-bit machine?

**Comment by duckartes** [comment]
Correct. Raspbian for Raspberry Pi 3 and Ubuntu to be specific.   I'll sync and test.

**Comment by duckartes** [comment]
Looks like that did it. It's indexing currently :)
